docs/reference/glossary.md

```diff
@@ -32,6 +32,16 @@ An image that has no parent is a **base image**.
 [boot2docker](http://boot2docker.io/) is a lightweight Linux distribution made
 specifically to run Docker containers. The boot2docker management tool for Mac and Windows was deprecated and replaced by [`docker-machine`](#machine) which you can install with the Docker Toolbox.
 
+## bridge network
+
+A bridge network is a Link Layer device (also called a Layer 2 device) which
```

Comment:
I agree with @thaJeztah . If the term in the glossay has to stay as `bridge network`, then we should write something like:

"The _bridge network_ is a docker network which makes use of a bridge to allow containers on that network to communicate with each other. Each container is directly connected to the bridge, and can reach to or be reached from the external network via the bridge interface."

Not sure if the details about what a bridge is belong here.